src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,16 @@
 package core.basesyntax;
 
-public class MainApp {
+import core.basesyntax.machines.Bulldozer;
+import core.basesyntax.machines.Excavator;
+import core.basesyntax.machines.Machine;
+import core.basesyntax.machines.Truck;
 
+public class MainApp {
+    static Machine[] machines = new Machine[]{new Bulldozer(), new Excavator(), new Truck()};
+    public static void main(String[] args) {
+        for (int i = 0; i < 3; i++){
```

Comment:
Consider using `machines.length` instead of hardcoding the loop limit to `3`. This will make the code more flexible and easier to maintain if the number of machines changes in the future.